### PR TITLE
Streamer shelves

### PR DESCRIPTION
**Pull request by PSYmoom**

### CODE REVIEW

File: GeorgeChampBot.py [hunk 1]
```diff
@@ -36,13 +32,12 @@
 
 @ut.client.event
 async def on_ready():
-    try:
```
Comment (mmaaz1) [inline GeorgeChampBot.py]: why is this deleted?

File: GeorgeChampBot.py [hunk 2]
```diff
@@ -240,13 +238,21 @@ async def print_help(message, message_content):
             !plsremove-dota <Name or Player ID> - Remove player from tracking list (Admin Only)
             !plslist-dota - List currently tracked players""")
         embed = ut.DiscordEmbedBuilder(colour_ = 0x0047AB, title_ = "Dota Commands", description_ = description, thumbnail_url = "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQp8emc_vN_kjb7616lE0JMIp9Igeko58cd1g&usqp=CAU")
+    elif message_content == "twitch":
+        description = inspect.cleandoc("""
+            !plsadd-twitch <Name> <Twitch Username> - Add streamer to tracking list (Admin Only)
+            !plsremove-twitch <Name or Twitch Username> - Remove streamer from tracking list (Admin Only)
+            !plslist-twitch - List currently tracked players""")
```
Comment (mmaaz1) [inline GeorgeChampBot.py]: nit: currently tracked streamers

File: GeorgeChampBot.py (hunk 1 above)
Comment (PSYmoom) [reply GeorgeChampBot.py]: Restored.

File: GeorgeChampBot.py (hunk 2 above)
Comment (PSYmoom) [reply GeorgeChampBot.py]: Fixed typo.
